docker/supabase/volumes/db/init/04-telescope-schema.sql

```diff
@@ -0,0 +1,41 @@
+-- Case-insensitive text type
+CREATE EXTENSION IF NOT EXISTS citext WITH SCHEMA extensions;
+
+-- Registered Telescope users
+CREATE TABLE IF NOT EXISTS telescope_profiles (
+  id text PRIMARY KEY,
+  registered_at timestamptz DEFAULT timezone('utc'::text, now()) NOT NULL,
+  updated_at timestamptz DEFAULT timezone('utc'::text, now()) NOT NULL,
+  display_name citext UNIQUE NOT NULL,
+  first_name text NOT NULL,
+  last_name text NOT NULL,
+  email citext UNIQUE NOT NULL,
+  github_username citext UNIQUE NOT NULL,
+  github_avatar_url text NOT NULL
+);
+
+-- Planet CDOT Feed List, see https://wiki.cdot.senecacollege.ca/wiki/Planet_CDOT_Feed_List
+CREATE TABLE IF NOT EXISTS feeds (
+  url text PRIMARY KEY,
+  user_id text references telescope_profiles ON DELETE CASCADE, -- optional, a user can claim an existing feed when they register
+  author_name text, -- owner of a feed, maybe unused when the feed is linked with an actual user
```

Comment:
Because `author_name` and `html_url` are only relevant for Wiki/legacy feeds, should we prefix the names (e.g., `wiki_author_name` and `wiki_html_url`)?  So that it's clear that these don't cross-over with the user info?

Reply:
@humphd, `wiki_author_name` sounds good, do we have the html_url (the main blog url) of a feed in the Planet Feed?